src/cityreader/cityreader.py

```diff
@@ -17,17 +22,22 @@
 cities = []
 
 def cityreader(cities=[]):
+ with open('cities.csv', newline='') as csvfile:
+  citylist = csv.reader(csvfile)
+  for row in citylist:
+   i = 1
```

Comment:
Check out the `next()` method. This could help you cut down on some code (using `i` to track what row you're on in the list).